### ISSUE TITLE
[S3] Enable CORS policy for bucket used by the FE Tools team

### ISSUE THREAD

**Issue by oseasmoran73** [department-of-veterans-affairs/va.gov-team#31528]
## Description
We would like to enable CORS policy for the following s3 bucket: `vetsgov-website-builds-s3-upload`

## Background/context 
[Ticket](https://app.zenhub.com/workspaces/vsp---frontend-tools-5fc9325744944e0015ed1861/issues/department-of-veterans-affairs/va.gov-team/29938) associated to request. 
Would like this policy enabled so we can use `vetsgov-website-builds-s3-upload/coverage/test-coverage-report.json` available in VFST dashboard.
Note: in speaking w/ @oseasmoran73 - When the FE Tools team tries to call the URL directly - to try to parse into the dashboard - they are unable to. 

---
## Tasks
- [ ] _What things need to happen?_

## Acceptance Criteria 
- [ ] _What are the specific things that must be true in order for this to be accepted?_

**Comment by omgitsbillryan** [comment]
Mind adding some screenshots? I'm not understanding the problem as described.

**Comment by jhouse-solvd** [comment]
@oseasmoran73 - Please see Bill's comment above. We'll need some add'l details before we can pick this up. Thank you.

**Comment by oseasmoran73** [comment]
Thanks for pinging @jhouse-solvd . Sometimes they get lost in emails 😅 . I will update ticket hopefully tomorrow before EOD. I recall @omgitsbillryan mentioning in office hours that some changes were made to the bucket and have not tested it out yet.

If changes are working on our end, I will close ticket out, otherwise update with screenshot. 

Sorry for the delay!

**Comment by oseasmoran73** [comment]
For transparency, there was something that was critical that required my attention. Have not had the chance to provide screenshot/verification on Wednesday or today.

I will close this ticket out for now, and once I get around to it, I will reopen / submit a new ticket. Thanks for being patient with me team!
